scripts/bump-changelogs.sh

```diff
@@ -2,7 +2,7 @@
 
 CHAP=./cardano-haskell-packages
 # Download a shallow copy of CHaP
-git clone --depth 1 git@github.com:IntersectMBO/cardano-haskell-packages.git $CHAP
+git clone --depth 1 https://github.com/IntersectMBO/cardano-haskell-packages $CHAP
```

Comment:
Why this change?

Reply:
You can alternatively fetch a tarball using the archive URL.

However, even better I think is to use the CHaP index file rather than the repo. Although the git repo and the package repo *should* never be out of step, using the index guarantees that you see exactly what Cabal sees. It also avoids having to run a script to calculate the package list from the repo state.

You can see an [example of doing this](https://github.com/IntersectMBO/cardano-haskell-packages/blob/main/scripts/check-cabal-plan.sh#L40) in a script I added to the CHaP git repo.

I think downloading the index file may also be a little faster because it's smaller than the whole repo.